src/commands/environmentVariables/editEnvironmentVariable/EnvironmentVariableEditDraftStep.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { type Container, type EnvironmentVar } from "@azure/arm-appcontainers";
+import { nonNullValue } from "@microsoft/vscode-azext-utils";
+import { type Progress } from "vscode";
+import { type ContainerAppItem } from "../../../tree/ContainerAppItem";
+import { type RevisionsItemModel } from "../../../tree/revisionManagement/RevisionItem";
+import { localize } from "../../../utils/localize";
+import { RevisionDraftUpdateBaseStep } from "../../revisionDraft/RevisionDraftUpdateBaseStep";
+import { type EnvironmentVariableEditContext } from "./EnvironmentVariableEditContext";
+
+export class EnvironmentVariableEditDraftStep<T extends EnvironmentVariableEditContext> extends RevisionDraftUpdateBaseStep<T> {
```

Comment:
It's not a huge deal, but I feel like this step could almost entirely be consolidated with the "AddStep".

But if you like having them separate just for organization/logical purposes, makes sense to me too.

Reply:
Yeah, although there are a lot of overlaps, there's enough differences that I'd prefer to leave them uncoupled to reduce complexity if we ever need to come back and alter the behavior.